examples/kubernetes/vttablet-pod-template.yaml

```diff
@@ -57,6 +57,7 @@ desiredState:
             -db-config-filtered-uname vt_filtered
             -db-config-filtered-dbname vt_{{keyspace}}
             -db-config-filtered-charset utf8
+            -queryserver-config-transaction-cap 300
```

Comment:
Sure, we'll have the potential problem of the two templates falling out of sync but I don't think that they'll be changed that frequently